samples/server/petstore/php-slim4/lib/Mock/OpenApiDataMocker.php

```diff
@@ -207,7 +228,72 @@ public function mockString(
             throw new InvalidArgumentException('"maxLength" value cannot be less than "minLength"');
         }
 
-        return str_pad('', mt_rand($minLength, $maxLength), 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ', \STR_PAD_RIGHT);
+        switch ($dataFormat) {
+            case IMocker::DATA_FORMAT_BYTE:
+            case IMocker::DATA_FORMAT_BINARY:
```

Comment:
I tried to produce binary string data with `random_bytes()` php function, but resulted string can break std output. I don't know how `binary` format string should look like(raw, hex. base64 etc.). If somebody knows, please give an example.

Reply:
What about using a random string generator mentioned in https://stackoverflow.com/a/4356295/677735 ?

> how binary format string should look like(raw, hex. base64 etc.).

Should be simply just raw.

Reply:
Function from link looks like a base62 encoded string to me. Raw binary string contains non UTF characters which breaks PHP output or even JSON.
I thought that raw binary string in php is:
```php
echo random_bytes(10);
// ????U???
```

Reply:
But your concern is that it may break the std output, right?

If that's not a concern, please start with `random_bytes`

Reply:
It's not my concern, I know that it breaks output :smile: I've checked. Maybe it should be encoded somehow to be presented inside of JSON or http request.